### PR TITLE
Add style on TextField

### PR DESCRIPTION
**Pull request by CCJY**
Maybe someone wants to customize style on textfield. Also, I need it.

### CODE REVIEW

**Comment by panuavakul** [comment]
@CCJY Thanks for the PR and sorry about my late reply.
LGTM, I will confirm it, merge and release new version asap

**Comment by joecks** [comment]
Could you please merge that, I need that too :)

File: lib/tag_editor.dart [hunk 1]
```diff
@@ -17,6 +17,7 @@ class TagEditor extends StatefulWidget {
     this.icon,
     this.enabled = true,
     // TextField's Props
+    this.textStyle = const TextStyle(),
```
Comment (panuavakul) [inline lib/tag_editor.dart:20]: I believe that `TextField` can handle `null` for style so I don't think you need the default value here.
```suggestion
    this.textStyle,
```
should be ok

**Review by panuavakul — APPROVED** [review]
I'm sorry again for my late action. I've been busy this past week.
I will try to release this today or tomorrow.

**Comment by panuavakul** [comment]
Just released to [0.0.5](https://pub.dev/packages/material_tag_editor). Thanks!

**Pull request merged**